c_glib/test/test-decimal.rb

```diff
@@ -52,4 +52,59 @@ def test_to_integer
     decimal = Arrow::Decimal128.new(integer_data)
     assert_equal(integer_data, decimal.to_i)
   end
+
+  def test_plus
+    integer_data1 = 23423445
+    integer_data2 = 5443
+    decimal1 = Arrow::Decimal128.new(integer_data1)
+    decimal2 = Arrow::Decimal128.new(integer_data2)
+    decimal3 = decimal1.plus(decimal2)
+    assert_equal(integer_data1 + integer_data2, decimal3.to_i)
+  end
+
+  def test_minus
+    integer_data1 = 23423445
+    integer_data2 = 5443
+    decimal1 = Arrow::Decimal128.new(integer_data1)
+    decimal2 = Arrow::Decimal128.new(integer_data2)
+    decimal3 = decimal1.minus(decimal2)
+    assert_equal(integer_data1 - integer_data2, decimal3.to_i)
+  end
+
+  def test_multiply
+    integer_data1 = 23423445
+    integer_data2 = 5443
+    decimal1 = Arrow::Decimal128.new(integer_data1)
+    decimal2 = Arrow::Decimal128.new(integer_data2)
+    decimal3 = decimal1.multiply(decimal2)
+    assert_equal(integer_data1 * integer_data2, decimal3.to_i)
+  end
+
+  def test_divide
+    integer_data1 = 23423445
+    integer_data2 = 5443
+    decimal1 = Arrow::Decimal128.new(integer_data1)
+    decimal2 = Arrow::Decimal128.new(integer_data2)
+    decimal3 = decimal1.divide(decimal2)
+    assert_equal(integer_data1 / integer_data2, decimal3.to_i)
+  end
+
+  def test_remainder
+    integer_data1 = 23423445
+    integer_data2 = 5443
+    decimal1 = Arrow::Decimal128.new(integer_data1)
+    decimal2 = Arrow::Decimal128.new(integer_data2)
+    decimal3 = decimal1.remainder(decimal2)
+    assert_equal(integer_data1 % integer_data2, decimal3.to_i)
+  end
+
+  def test_divide_zero
+    decimal1 = Arrow::Decimal128.new(23423445)
+    decimal2 = Arrow::Decimal128.new(0)
+    message =
+      "[decimal][divide]: Invalid: Division by 0 in Decimal128"
+    assert_raise(Arrow::Error::Invalid.new(message)) do
+      decimal1.divide(decimal2)
+    end
+  end
```

Comment:
Can you add `remainder` with zero case too?

Reply:
Thanks! I'll unify `divide()` and `remainder()`.